src/controller/python/chip/FabricAdmin.py

```diff
@@ -58,50 +62,44 @@ class FabricAdmin:
 
         >> Persistence
 
-        Each instance persists its fabric ID and index to storage. This is in addition to the
-        persistence built into the ExampleOperationalCredentialsIssuer that persists details
+        Specifically, each instance persists its fabric ID and admin
+        index to storage. This is in addition to the persistence built into the ExampleOperationalCredentialsIssuer that persists details
         about the RCAC/ICAC as well. This facilitates re-construction of a fabric admin on subsequent
         boot for a given fabric and ensuring it automatically picks up the right ICAC/RCAC details as well.
     '''
 
-    _isActive = False
-    activeFabricIndexList = set()
-    activeFabricIdList = set()
+    activeAdminIndexList = set()
     activeAdmins = set()
-    vendorId = None
 
     @classmethod
     def _Handle(cls):
         return chip.native.GetLibraryHandle()
 
-    def AllocateNextFabricIndex(self):
-        ''' Allocate the next un-used fabric index.
+    def AllocateNextAdminIndex(self):
+        ''' Allocate the next un-used admin index.
         '''
-        nextFabricIndex = 1
-        while nextFabricIndex in FabricAdmin.activeFabricIndexList:
-            nextFabricIndex = nextFabricIndex + 1
-        return nextFabricIndex
+        nextAdminIndex = 1
+        while nextAdminIndex in FabricAdmin.activeAdminIndexList:
+            nextAdminIndex = nextAdminIndex + 1
+        return nextAdminIndex
 
-    def AllocateNextFabricId(self):
-        ''' Allocate the next un-used fabric ID.
-        '''
-        nextFabricId = 1
-
-        while nextFabricId in FabricAdmin.activeFabricIdList:
-            nextFabricId = nextFabricId + 1
-        return nextFabricId
-
-    def __init__(self, vendorId: int, rcac: bytes = None, icac: bytes = None, fabricIndex: int = None, fabricId: int = None):
+    def __init__(self, vendorId: int, rcac: bytes = None, icac: bytes = None, adminIndex: int = None, fabricId: int = 1):
         ''' Creates a valid FabricAdmin object with valid RCAC/ICAC, and registers itself as an OperationalCredentialsDelegate
             for other parts of the system (notably, DeviceController) to vend NOCs.
 
             vendorId:       Valid operational Vendor ID associated with this fabric.
             rcac, icac:     Specify the RCAC and ICAC to be used with this fabric (not-supported). If not specified, an RCAC and ICAC will
                             be automatically generated.
 
-            fabricIndex:    Local fabric index to be associated with this fabric. If omitted, one will be automatically assigned.
-            fabricId:       Local fabric ID to be associated with this fabric. If omitted, one will be automatically assigned.
+            adminIndex:     Local index to be associated with this fabric. This is NOT the fabric index. Each controller on the fabric
```

Comment:
```suggestion
            adminIndex:     Local index to be associated with this admin/CA. This is NOT the fabric index. Each controller on the fabric
```

Reply:
This is not the ID associated with the admin / CA - it truly is associated with the fabric.